src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -166,9 +168,9 @@ public function format($message)
             }
         }
 
-        $output .= $this->applyCurrentStyle(substr($message, $offset));
+        $output .= $this->applyCurrentStyle(mb_substr($message, $offset, null, 'ASCII'));
```

Comment:
8bit

Reply:
I agree it should be `8bit` but The symfony mbstring polyfill component uses `ASCII` as the identifier for `8bit`.

Using ASCII lets the code work in Symfony for systems without the mbstring extension.